### PR TITLE
Update the default template-validator image to the latest released one

### PR DESCRIPTION
**Pull request by omeryahud**
**What this PR does / why we need it**:
Update the template validator image deployed by the SSP operator to the
latest released one on every release.

**Which issue(s) this PR fixes**: 

The default template validator image should no longer be manually updated for each release

**Release note**:

```release-note
NONE
```
/cc @akrejcir @kwiesmueller

### CODE REVIEW

**Comment by akrejcir** [comment]
Related to: https://github.com/kubevirt/ssp-operator/issues/178

**Review by akrejcir — CHANGES_REQUESTED** [review]
The `Dockerfile` needs to be updated too.
The `make manager` target is probably not used anywhere. Maybe only for local building.

Maybe we can call makefile targets directly from the `Dockerfile` ?

**Comment by omeryahud** [comment]
> The `Dockerfile` needs to be updated too.
Ah right, I assumed it was used in the Dockerfile, I will update it as well

> The `make manager` target is probably not used anywhere. Maybe only for local building.
> 
> Maybe we can call makefile targets directly from the `Dockerfile` ?
Sounds like a good idea,  I will check it out

**Comment by omeryahud** [comment]
@akrejcir Can you take another look?

File: Dockerfile [hunk 1]
```diff
@@ -8,24 +8,25 @@ COPY go.sum go.sum
...
 COPY api/ api/
 COPY controllers/ controllers/
 COPY internal/ internal/
 
-COPY hack/csv-generator.go csv-generator.go
+COPY hack/csv-generator.go hack/csv-generator.go
 
 # Build
-RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -o manager main.go
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on make manager
```
Comment (akrejcir) [inline Dockerfile:20]: Maybe the docker argument needs to be passed here as en env variable to make? I haven't checked it, so I'm not sure.

File: Makefile [hunk 2]
```diff
@@ -106,7 +108,14 @@ container-build: unittest bundle
 	mkdir -p data/crd
 	cp bundle/manifests/ssp-operator.clusterserviceversion.yaml data/olm-catalog/ssp-operator.clusterserviceversion.yaml
 	cp bundle/manifests/ssp.kubevirt.io_ssps.yaml data/crd/ssp.kubevirt.io_ssps.yaml
-	docker build -t ${IMG} .
+	docker build -t ${IMG} \
+		--build-arg IMG_REPOSITORY=${IMG_REPOSITORY} \
```
Comment (akrejcir) [inline Makefile:112]: Do these build arguments need to be defined inside the Dockerfile using `ARG` instruction?

Comment (omeryahud) [reply Makefile:112]: Yes, you are right, I thought it was implicit, but it isn't, fixed now

File: Dockerfile (hunk 1 above)
Comment (omeryahud) [reply Dockerfile:28]: It is enough that the env vars are consumed with `ARG`, so it should be good now

Comment (akrejcir) [reply Dockerfile:28]: Ok

**Review by akrejcir — COMMENTED** [review]
/lgtm

**Comment by akrejcir** [comment]
CI fails, because there was an issue with latest kubevirt release. They are working on it.

**Comment by akrejcir** [comment]
New kubevirt version was released. 

/retest

**Pull request merged**